src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day--sun calendar--month-length">
```

Comment:
return default modifier value in here
```suggestion
    <div class="calendar calendar--start-day--sun calendar--month-length--29">
```

Reply:
Я ничего не поняла. Просьба дать пояснения. Почему нельзя оставить так, как сделано? Оно работает, я проверяла. Если менять количество дней - оно и меняется.